src/odemis/driver/xt_client.py

```diff
@@ -790,6 +790,77 @@ def set_use_case(self, state):
             self.server._pyroClaimOwnership()
             return self.server.set_use_case(state)
 
+    def get_mpp_orientation(self):
+        """
+        Get the current multi probe orientation in degrees.
+
+        :return (float): Multi orientation in degrees
```

Comment:
For the stage rotation, the values reported are in radians. Are you sure it's here in degrees? 
Also, here, the range would be useful.

Reply:
Yes TFS returns it in degrees, this should be a by designed defined and set value.
The range is kind of hard to add here. It should be -18 to 18 degrees.
However, due to hardware constrictions it is currently 45 degrees turned, an now defined at 45 degrees. This will stay this  value as it cannot change (unless the internal hardware of the microscope changes).

Reply:
The -18 to 18 degrees range was on FumoBeta where no k-mirror was available. On EA it should be -45 to 45 degrees to keep the beam index in the pattern well defined (due to the 4-fold symmetry).  What values did xt return for the range of the mmp? I see that in your tests you check that the angle is -90 < angle < 90 degrees. If it is indeed 90 degrees I need to check with TFS.